### ISSUE TITLE
Community development

### ISSUE THREAD

**Issue by dimontau** [PiBrewing/craftbeerpi4#114]
Greetings Alexander. Maybe it's time to create a community for this project. So as not to disturb you, on minor issues, but decide for yourself? For example, in Telegram?

**Comment by avollkopf** [comment]
@dimontau there is already the plan to use discord. @prash3r already started something. The only open item is sort of a kick-off where I am the limiting factor due to time constraints. But you are right, It's time to start this as topic discussions within FB are not really valuable.

**Comment by prash3r** [comment]
Yes, i created a discord as discord seems ideal for this purpose with different channels etc. @avollkopf joined it already and is admin and can create invitation links.

I have setup an example structure of channels and permissions to have a basis on which to discuss changes.

![image](https://github.com/PiBrewing/craftbeerpi4/assets/4346028/6e02ff76-04ff-409f-bbc8-6a8cd3971285)

I would love to have some sort of internal kickoff meeting where we can meet with preferably as many active devs as possible to discuss channel and permission structure and default permissions before sharing the invite link with the community and adding it to repos and source and documentations. We also need to decide if we rename it from craftbeerpi to PiBrewing before publicly announcing it, because i created this before the org was formed.

**I would also be happy to have this discussion not as a meeting but as channel inside the discord itself as some kind of test/evaluation run to see if discord fits our needs and everyone can partake at their own time.**

Since Alexander didn't share the invite link publicly i wont either. You can DM me on discord (also prash3r) if you want to have a look and ill invite you.

**Comment by berry-langerak** [comment]
Any update on this? I would happily get involved in development, but I don't know how?

**Comment by prash3r** [comment]
I just had 2 people take a look at the empty server and tweaked some settings with them. It should be alright to make the invite link public.

[PiBrewing discord server invite link](https://discord.gg/UtjQ3kgJY6) (EDITED: updated link to to never expire)

If you are not familliar with discord make sure you check out the #readme channel on the server to see how to selectively mute notifications and not get notifications on all messages in all channels. Or just enable Notifications for the channels that you are interested in. Otherwise it can get annoying very quickly.

If there are no objections i will write pull requests this month to add the discord link with logo to the github readme.md files of [PiBrewing/craftbeerpi4](https://github.com/PiBrewing/craftbeerpi4) and [PiBrewing/craftbeerpi4-ui](https://github.com/PiBrewing/craftbeerpi4-ui). should i also add a logo-link to the [PiBrewing/craftbeerpi4-ui](https://github.com/PiBrewing/craftbeerpi4-ui) project itself as well so users can join the discord directly from their own cbpi4 instance web ui or should we postpone that until we actually have a community on the server?

**Comment by alxhslm** [comment]
Bumping on this. I would also be keen to get involved in development 

Discord sounds perfect but @prash3r your link has expired.

**Comment by prash3r** [comment]
I updated the discord invite link to never expire

[PiBrewing discord server invite link](https://discord.gg/UtjQ3kgJY6) (EDITED: updated link to to never expire)
